fabric/forgero-fabric-compat/src/main/resources/assets/forgero/lang/en_us.json

```diff
@@ -1,21 +1,71 @@
 {
-  "forgero.menu.options": "Forgero configuration",
-  "forgero.menu.options.error": "Error...",
-  "forgero.menu.options.reset": "Reset",
-  "forgero.menu.options.true": "True",
-  "forgero.menu.options.false": "False",
-  "forgero.menu.options.reload_config": "Reload config from disk",
-  "forgero.menu.options.disabledResources": "Disabled resources",
-  "forgero.menu.options.disabledPacks": "Disabled packs",
-  "forgero.menu.options.disableVanillaRecipes": "Disable vanilla recipes",
-  "forgero.menu.options.disableVanillaLoot": "Disable vanilla loot",
-  "forgero.menu.options.disableVanillaTools": "Disable vanilla tools",
-  "forgero.menu.options.convertVanillaRecipesToForgeroTools": "Vanilla recipes to Forgero tools",
-  "forgero.menu.options.convertVanillaToolLoot": "Convert vanilla tool loot",
-  "forgero.menu.options.enableUnbreakableTools": "Enable unbreakable tools",
-  "forgero.menu.options.enableRepairKits": "Enable repair kits",
-  "forgero.menu.options.resourceLogging": "Resource logging",
-  "forgero.menu.options.logDisabledPackages": "Log disabled packages",
-  "forgero.menu.options.useEntityAttributes": "Entity attributes",
-  "forgero.menu.options.baseSoulLevelRequirement": "Base soul level"
+  "forgero.config.title": "Forgero Configuration",
+  "forgero.config.category.main.title": "Main",
+  "forgero.config.category.main.title.tooltip": "Forgero's main settings.",
+  "forgero.config.disabledResources": "Disabled resources",
+  "forgero.config.disabledResources.tooltip": "Resources that shouldn't be loaded by Forgero.",
+  "forgero.config.disabledPacks": "Disabled packs",
+  "forgero.config.disabledPacks.tooltip": "Packs that shouldn't be loaded by Forgero.",
+  "forgero.config.group.recipes_upgrades_and_loot.title": "Recipes, upgrades, and loot",
+  "forgero.config.group.recipes_upgrades_and_loot.title.tooltip": "Configuration for recipes, upgrades, and loot.",
+  "forgero.config.disableVanillaRecipes": "Disable vanilla recipes",
+  "forgero.config.disableVanillaRecipes.tooltip": "Disables Minecraft's vanilla tool recipes.",
+  "forgero.config.enableCustomRecipeDeletion": "Enable custom recipe deletion",
+  "forgero.config.enableCustomRecipeDeletion.tooltip": "Enables deletion of recipes via custom IDs, using a datapack system.",
+  "forgero.config.enableRecipesForAllSchematics": "Enable recipes for all schematics",
+  "forgero.config.enableRecipesForAllSchematics.tooltip": "Enables recipes for all schematics, making all schematics craftable.",
+  "forgero.config.enableUpgradeInCraftingTable": "Enable upgrade in crafting table",
+  "forgero.config.enableUpgradeInCraftingTable.tooltip": "Enables upgrading tools in a crafting table, for example, from wood to metal.",
```

Comment:
This setting actually makes it possible to upgrade parts and tools in the crafting table, not just the smithing table. But this really is a deprecated setting.